src/bussinessLogic/ManageArray.java

```diff
@@ -31,34 +35,36 @@
     private String charString;
     // current char
     private char currentChar;
-    // hash map for storing and recording the sorted chars
-    private LinkedHashMap<String, Integer> sortedMap;
 
+    public ManageArray(MapManager manageMap) {
```

Comment:
Please add a comment to this constructor.

Reply:
👍